src/clients/github.js

```diff
@@ -159,14 +185,15 @@ async function createSourceFileInRepo(
   fileName,
   source,
 ) {
-  await performWithRetryNetworkErrors(() => github.getRepo(userName, repoName).
-    writeFile(
-      branchName,
-      fileName,
-      source,
-      COMMIT_MESSAGE,
-      {},
-    ),
+  await performWithRetryNetworkErrors(
+    () => github.getRepo(userName, repoName).
+      writeFile(
+        branchName,
+        fileName,
+        source,
+        COMMIT_MESSAGE,
+        {},
+      ),
```

Comment:
Probably not relevant for this PR, but I did just come across [a way to create/update multiple files via the GitHub API with a fixed number of API requests](https://gist.github.com/StephanHoyer/91d8175507fcae8fb31a).

As it happens, that number of API requests is probably not meaningfully smaller than the number we make in the current implementation, so probably mostly of academic interest unless we find ourselves increasing the number of files we write to GitHub…